docs/setup/pi4humble.md

```diff
@@ -0,0 +1,94 @@
+
+## Step-by-step
+
+1. Download and install the appropriate version of the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) on your computer.
+
+1. Open the imager and under operating system, select "Other general-purpose OS" then "Ubuntu" and finally, "Ubuntu Server 22.04 LTS (64-bit)".
+
+1. Insert your microSD card into your computer and under storage in the Raspberry Pi Imager, select your microSD card. 
+
+1. Select the gear icon to optionally customize advanced options including enabling SSH, setting a unique username and password, and configuring a wireless LAN. 
+
+1. In the system-boot partition, edit config.txt and add `dtoverlay=dwc2,dr_mode=peripheral` at the end of the file.
+
+1. In the system-boot partition, edit cmdline.txt to add `modules-load=dwc2,g_ether` after `rootwait`.
+
+1. In the system-boot partition, edit the `network-config` executable to optionally add information about your Wi-Fi connection, and also add the following under `ethernets`
+
+                usb0:
+                    dhcp4: false
+                    optional: true
+                    addresses: [192.168.186.3/24]
+
+    For convenience, [here's a copy of this file](data/network-config.txt). Be sure to remove the `.txt` extension.
+    Note that the robot uses the default IP address of 192.168.186.2 on its usb0 interface.
+    Please note also that after initial boot, editing `network-config` in the boot partition will not do anything; instead, you will need to edit `/etc/netplan/50-cloud-init.yaml`.
+    After editing that file, you will need to run `sudo netplan generate` followed by `sudo netplan apply` to update your network configuration.  
+
+1. If you would like your Raspberry Pi® 4[^3] to communicate with the Create® 3 over its USB-C®[^2] port (and not just to power it), be sure that the [USB/BLE toggle on the robot's adapter board](../../hw/electrical/#adapter-board-overview) is set to the USB position.
+
+1. Insert the microSD card into the Raspberry Pi® 4[^3], and then use a USB-C®[^2] to USB-C®[^2] cable to connect the Raspberry Pi® 4[^3] to the Create® 3.
+A photo of this connection can be found [here](../../hw/hookup/#raspberry-pi-4).
+The first boot may take a few minutes. (It may help to have a monitor and keyboard set up in case of any trouble on the first boot.)
+
+!!! attention
+        If you are new to ROS 2 and would like to utilize the turtlesim tutorials, it necessary to install a desktop environment to do this. 
+        We have had success with xubuntu which can be installed with the following command: `sudo apt install xubuntu-desktop`. 
+        Please see the [Ubuntu docs](https://ubuntu.com/tutorials/how-to-install-ubuntu-on-your-raspberry-pi#5-install-a-desktop) for more details. 
+
+
+10. Log in with the default username and password (ubuntu/ubuntu), change your password, and then change your locale to be one that uses UTF-8. For example, in the US, type
```

Comment:
Something happened to the numbering here after the admonition (in mkdocs it gets reset to 1). I have run into this before and it took some fiddling to figure it out, lmk if you want me to play with it

Reply:
Yes, I think I figured it out now. Should be good to go!